manifests/hardware.json

```diff
@@ -19,6 +19,10 @@
             },
             "gpuModel": {
                 "title": "GPU Model",
+                "description": [
+                    "Share of primary Models GPUs.",
```

Comment:
@SuYoungHong I'm not sure if I understand this. Is there a different way to phrase this?

Reply:
I think this was a typo on my part. I meant to say, 

`"Share of primary GPU models"`

Does that make more sense?